### ISSUE TITLE
Add a recaptcha option for forms.

### ISSUE THREAD

**Issue by netzih** [progressivetech/net.ourpowerbase.remoteform#15]
Not sure if this belongs in issues as it's a feature request, or if it's even possible. 
But is there a way to add the recaptcha for the form widget? 

Thank you.

**Comment by jmcclelland** [comment]
Yes, feature requests are welcome! I'm not sure I can get to this anytime soon, but it seems like a very good feature request. 

I think the web site owner would have to add a new javascript call to include the recaptcha code from Google's server. However, after that, it would be nice if remoteform could simply pick up on whether the profile is configured to require captcha or not. If it is required, then remoteform would both display the recaptcha form and reject any post that didn't come with a valid recaptcha token.

**Comment by nillde** [comment]
This feature would also help me a lot!
